src/App.tsx

```diff
@@ -1,93 +1,160 @@
-              <input type="checkbox" className="toggle" id="toggle-view" />
-              <label htmlFor="toggle-view">asdfghj</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="completed">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-completed" />
-              <label htmlFor="toggle-completed">qwertyuio</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-editing" />
-              <label htmlFor="toggle-editing">zxcvbnm</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view2" />
-              <label htmlFor="toggle-view2">1234567890</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
-      </section>
-
-      <footer className="footer">
-        <span className="todo-count" data-cy="todosCounter">
-          3 items left
-        </span>
-
-        <ul className="filters">
-          <li>
-            <a href="#/" className="selected">All</a>
-          </li>
-
-          <li>
-            <a href="#/active">Active</a>
-          </li>
-
-          <li>
-            <a href="#/completed">Completed</a>
-          </li>
-        </ul>
-
-        <button type="button" className="clear-completed">
-          Clear completed
-        </button>
-      </footer>
+        </>
+      </Header>
+
+      {todos.length > 0 && (
+        <>
+          <Main>
+            <TodoList>
+              <>
+                {visibleTodos.map(todo => {
+                  const { id, title, completed } = todo;
+
+                  return (
+                    <TodoCard
+                      key={id}
+                      id={id}
+                      title={title}
+                      completed={completed}
+                      todosUpdater={todosUpdater}
+                      todos={todos}
+                    />
+                  );
+                })}
+              </>
+            </TodoList>
+          </Main>
+
+          <Footer>
+            <>
+              <TodoCounter todos={todos} />
+
+              <Filters
+                filterChange={filterChange}
+                filter={filter}
+              />
+
+              {Boolean(checkCompletedTodo.length) && (
```

Comment:
Also, you could use logical operators
```suggestion
              {!!checkCompletedTodo.length && (
```